### PR TITLE
Configuration article (webiny.root.js & webiny.config.js)

### LINKED ISSUE #86: Configuration files
We need to explain what `webiny.root.js` and `webiny.config.js` do and how to configure them

### PR DESCRIPTION
**Pull request by Fsalker**
Here's an article explaining `webiny.root.js` & `webiny.config.js`

resolves #86

### CODE REVIEW

File: docs/get-started/configuration.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+sidebar_label: Configuration
+---
+
+There are two files you can configure right now: `webiny.root.js` and `webiny.config.js`
+
+## webiny.root.js
+
+This file appears in your project's root and it is unique. These are the configurable fields:
+
+- **projectName** - your project's name; it is passed into the **Context** the plugins receive and it is also used when delpoying to AWS.
```
Comment (EmilK15) [inline docs/get-started/configuration.md:13]: delpoying -> deploying

File: docs/get-started/configuration.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+- **cli**
+    -  **plugins** - an array of CLI and Scaffold plugins that are usable in our CLI
```
Comment (EmilK15) [inline docs/get-started/configuration.md:15]: Maybe an example of a plugin, or a link to something that already references how to create a plugin?

File: docs/get-started/configuration.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+## webiny.config.js
+
+This file appears in various packages. It serves two purposes: attaching **commands** to `webiny run [command]` and adding deployment **hooks**. These are the configurable fields:
+- **commands** - an array of functions that are added to `webiny run [command]`. A notable example is importing **buildFunction** from `@webiny/serverless-files/functions/download/bundle` and passing it as the **build** command in order to run `webiny run build`.
+- **hooks** 
+    -  **stateChanged** - an array of hooks ran after deploying components to AWS.
```
Comment (EmilK15) [inline docs/get-started/configuration.md:22]: maybe explain a bit about the "array of hooks". I don't really get what these mean/do.

**Comment by Pavel910** [comment]
I'll edit this article myself, so don't merge it just yet. It needs better explanations and more context.

**Pull request merged**